src/Protobuf/Decode.elm

```diff
@@ -726,16 +777,46 @@ tagDecoder =
             )
 
 
-varIntDecoder : Decode.Decoder ( Int, Int )
-varIntDecoder =
+pack : IntOperations int -> Decode.Decoder ( Int, int )
```

Comment:
I find the `pack` and `packWith` names confusing. We are unpacking here.
`pack` is just `varIntDecoder`? And `packWith` is a mapped `varIntDecoder`?

Reply:
for `bool` it could just be
```
    varIntDecoder int32Operations
        |> Decode.map (Tuple.mapSecond ((/=) 0))
        |> packedDecoder VarInt
```

Reply:
Good point, I removed the pack/packWith methods. Especially pack was just an varIntDecoder alias *facepalm*